app/styles/tracker.scss

```diff
@@ -262,6 +263,13 @@ dex {
       cursor: pointer;
       background-color: $brand-secondary-dark;
     }
+
+    &.visible {
+      @include transition(all .5s ease-in);
```

Comment:
i don't think you need this `@include` - just the one under `.scroll-up` is fine